Bundle/SymfonyBridge/src/DependencyInjection/Compiler/AutoRegister.php

```diff
@@ -58,6 +58,10 @@ public function process(ContainerBuilder $container)
                         continue;
                     }
 
+                    if ($parameters[0]->getClass() === null) {
+                        throw new \Exception(sprintf('Could not get register class %s', $method->class));
```

Comment:
Can you change this to a RuntimeException and improve the text a bit?

```suggestion
                        throw new \RuntimeException(sprintf('Could not get auto register class %s because the first parameter %s of public method %s should be have a class typehint. Either specify the typehint, make the function non-public, or disable auto registration.', $method->class, $parameters[0]->getName(), method->getName()));
```